main/src/ca/uwaterloo/flix/language/phase/Kinder.scala

```diff
@@ -968,7 +969,7 @@ object Kinder {
       val pvar = Type.freshVar(Kind.Eff, m.loc)
       // Note: We must NOT use [[Validation.toSoftFailure]] because
       // that would duplicate the error inside the Validation.
-      Validation.SoftFailure(KindedAst.Expr.Error(m, tvar, pvar), LazyList.empty)
+      Validation.SoftFailure(KindedAst.Expr.Error(m, tvar, pvar), Chain.empty)
```

Comment:
This is using the constructor directly, but I guess we will deal with that later and separately.